styles/components/grid.css

```diff
@@ -0,0 +1,21 @@
+.grid-container-2{
+    max-width: 800px;
+    border:1px solid var(--border-color)
+    
+}
+.grid-container-3{
+    max-width: 800px;
+    border:1px solid var(--border-color)
+    
+}
```

Comment:
You can extract out the repeating code inside a single class. It will work same.